Drift/Private/DriftBase.h

```diff
@@ -114,10 +114,11 @@ class FDriftBase : public IDriftAPI, public FTickableGameObject
     void UpdateFriendsList() override;
     bool GetFriendsList(TArray<FDriftFriend>& friends) override;
     FString GetFriendName(int32 friendID) override;
-    bool IssueFriendToken(int32 PlayerID, const FDriftIssueFriendTokenDelegate& delegate) override;
+    bool IssueFriendToken(int32 PlayerID, FDriftFriendTokenProperties TokenProperties, const FDriftIssueFriendTokenDelegate& delegate) override;
     bool AcceptFriendRequestToken(const FString& token, const FDriftAcceptFriendRequestDelegate& delegate) override;
     bool DeclineFriendRequest(int32 RequestId, FDriftDeclineFriendRequestDelegate& delegate) override;
     bool GetFriendRequests(const FDriftGetFriendRequestsDelegate& Delegate) override;
+	bool GetFriendInvites(const FDriftGetFriendRequestsDelegate& Delegate) override;
```

Comment:
`GetSentFriendInvites` might be clearer?

Reply:
Maybe.

With context of GetFriendRequests and the comment, I think it's ok, but in isolation I agree it can be misleading.
Want me to change it?

Reply:
yes please, context is always lost